packages/ssi-service/src/nav/nav.css

```diff
@@ -172,6 +172,7 @@
 .op-nav__icon {
   display: block;
   font-size: 1.25rem;
```

Comment:
You can remove the font-size as we're using the svg icon instead of the font icon now.

Reply:
@deshmukhmayur Correct. The changes have been made can you please review?